config/locales/en.yml

```diff
@@ -2520,8 +2520,15 @@ en:
       title: Edit User
     edit:
       title: Edit User
-      discord_title: Discord
-      discord_description: "This will help you gain access to locked channels in discord"
+      discord_connect: Join Our Discord Community
+      discord_title: Join Our Discord Server
+      discord_description: Joining our Discord server will let you get in touch with our team, coaches, and your peers instantly.
+      discord_how_it_works_title: How it works
+      discord_how_it_works: <li> Join our Discord server (it's free!) </li><li>Link your Discord account with this school's account.</li><li>Gain access to our community on Discord!</li>
```

Comment:
This should work with `_html` key suffix. There's more HTML translations in here that could be updated as well.